lib/chef/knife/client_show.rb

```diff
@@ -42,6 +42,7 @@ def run
 
         client = Chef::ApiClientV1.load(@client_name)
         output(format_for_display(client))
+        self.class.attrs_to_show = []
```

Comment:
I see we are doing this in environment_show and other commands as well.  Why is it it needed, isn't this command about to exit right after we call this?  Perhaps a comment would be appropriate here?